CHANGELOG.md

```diff
@@ -29,14 +29,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#216](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/216))
 - `opentelemetry-instrumentation-grpc` Add tests for grpc span attributes, grpc `abort()` conditions
   ([#236](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/236))
+- Add README and example app for Prometheus Remote Write Exporter
+  ([#227](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/227]))
 
 ### Changed
 - `opentelemetry-instrumentation-asgi`, `opentelemetry-instrumentation-wsgi` Return `None` for `CarrierGetter` if key not found
   ([#1374](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/233))
 - `opentelemetry-instrumentation-grpc` Comply with updated spec, rework tests
   ([#236](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/236))
 - `opentelemetry-instrumentation-asgi`, `opentelemetry-instrumentation-falcon`, `opentelemetry-instrumentation-flask`, `opentelemetry-instrumentation-pyramid`, `opentelemetry-instrumentation-wsgi` Renamed `host.port` attribute to `net.host.port`
-  ([#242](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/242))  
+  ([#242](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/242))
```

Comment:
Same for this white space.

Reply:
The whitespace is being removed, not added 😄